### DISCUSSION TITLE
Center align Header title

### DISCUSSION THREAD

**Discussion by debjanipaul** [h2oai/wave#1544] [Q&A]
Hi, is there a way I could center align the Header title for the header_card component. Also, is it possible to style individual UI components in wave? I was trying to change the style of the Header but doing so, it also transforms the footer component as they share the same CSS class in wave.

```py
q.page['header'] = ui.header_card(
        box='header',
        title='Some Dashboard',
        image=image,
        subtitle='',
        items=[
            ui.text('Dashboards', name='dashboard_title'),
            ui.menu(name='Dashboards', items=dashboards)
        ],
    )
```

**Comment by mturoci** [comment]
Hi @debjanipaul, good question.

>Also, is it possible to style individual UI components in wave

It is possible via Custom CSS, but not recommended. We try to take care of everything for you. If you have objective reasons on how some components could be improved, feel free to file a feature request. If the problem is more of a "cosmetic issue" (like alignment), read on.

> I was trying to change the style of the Header but doing so, it also transforms the footer component as they share the same CSS class in wave.

CSS class names are autogenerated by our build tooling which makes them inappropriate for CSS selectors. Instead, you can notice there are `data-test` attributes all over the HTML markup. These are filled based on either card name (`q.page['<card-name>']`) or on the `name` attribute in case of components. It's recommended to use `data-test` attributes where possible instead.

For your particular case, that would be:

```py
from h2o_wave import main, app, Q, ui


@app('/')
async def serve(q: Q):
    # Header spec.
    q.page['my_header'] = ui.header_card(box='1 1 -1 1', title='Title', subtitle='Subtitle')
    # Change CSS.
    css = '''
div[data-test="my_header"]>div {
  flex-grow: 1;
  justify-content: center;
}
'''
    q.page['meta'] = ui.meta_card(box='', stylesheet=ui.inline_stylesheet(css))

    await q.page.save()

```
<img width="1678" alt="image" src="https://user-images.githubusercontent.com/64769322/179205129-c72b5a5e-3425-444e-bb0b-128ae43f4b17.png">

Will also add this to our docs.
